### PR TITLE
Tutorials Update

### PR DESCRIPTION
**Pull request by blink1073**
cc @willingc @jasongrout I took a pass through the tutorials.

### CODE REVIEW

File: tutorial/terminology.md [hunk 1]
```diff
@@ -6,8 +6,10 @@ to help smooth the learning the components.
...
-- Phosphor
-- Standalone example
-- TypeScript
+- Application - The main application object that hold the application shell,
+command registry, and keymap registry.  It is provided to all plugins in their
+activate method.
+- Plugin - An object that provides a service and or extends the application.
+- Phosphor - The JavaScript library that provides the foundation of JupyterLab, enabling desktop-like applications in the browser.
+- Standalone example - An example in the `examples/` directory that demonstrates the usage of one or more components of JupyterLab.
+- TypeScript - A staticly typed language that compiles to JavaScript.
```
Comment (jasongrout) [inline tutorial/terminology.md]: statically

File: tutorial/adding_content.md [hunk 2]
```diff
@@ -11,9 +11,9 @@ files: `npm install -g tsd`.
 - If there are no typings, we must create our own.  An example typings file that exports functions is [ansi_up](https://github.com/jupyter/jupyterlab/blob/master/typings/ansi_up/ansi_up.d.ts).  An example with a class is [xterm](https://github.com/jupyter/jupyterlab/blob/master/typings/xterm/xterm.d.ts).
 - Add a reference to the new library in `src/typings.d.ts`.
 - Create a folder in `src` for the plugin.
-- Add  `widget.ts` and `plugin.ts` files.
-- Optionally, add the plugin to default themes (e.g. `src/default-themes/leaflet.less`), and import that theme in `src/default-themes/index.less`.
-- The `widget.ts` file should have the core logic for the plugin.  In
+- Add  `index.ts` and `plugin.ts` files.
+- If creating CSS, import the them in `src/default-themes/index.css`.
```
Comment (vidartf) [inline tutorial/adding_content.md]: "the them" --> "them"?

Comment (blink1073) [inline tutorial/adding_content.md]: Fixed, thanks!

**Comment by jasongrout** [comment]
Great improvements - thanks!

**Comment by willingc** [comment]
Thanks @blink1073 and @jasongrout :sunny:

**Pull request merged**